d2l-table-shared-styles.js

```diff
@@ -1,24 +0,0 @@
-import '@polymer/polymer/polymer-legacy.js';
```

Comment:
Only referenced by `d2l-table-style.js`.

Reply:
A bit confused how this is working... how are these set in the Polymer case now?  We just import the [`tableStyles` from the Lit version](https://github.com/BrightspaceUI/core/blob/master/components/table/table-wrapper.js#L6), but don't get the [defaults set in `styles`](https://github.com/BrightspaceUI/core/blob/master/components/table/table-wrapper.js#L203) right?

Reply:
Ah nvm - hadn't gotten to `d2l-table` yet, where these are added

Reply:
Yeah like in Polymer-land they'll either be using `<d2l-table-wrapper>` which will load the Lit stuff and define the defaults on the wrapper and include these global styles, OR they'll be using `<d2l-table>` which defines the defaults on itself and uses the style module.